src/test/java/com/adivinaLaPalabra/squad2/back/AdivinaLaPalabra/controllers/GameControllerTest.java

```diff
@@ -46,49 +44,53 @@ public class GameControllerTest {
     @Mock
     WordRepository wordRepository;
 
-    @Autowired
-    ObjectMapper objectMapper;
-
     @Test
     void testEndpointNewGameMustReturnOK() throws Exception {
         when(gameService.newGame()).thenReturn(GAME);
-        this.mockMvc.perform(MockMvcRequestBuilders.get(NEW_GAME_URL))
+        this.mockMvc.perform(MockMvcRequestBuilders.get(NEW_GAME_URL)
+                .header("Authorization", "Bearer " + AUTH_TOKEN))
```

Comment:
¿No os sirve lo de `@WithMockUser(value = "spring")` que os comenté? Así os ahorráis meter la cabecera en cada llamada.